functional-test/src/test/groovy/org/rundeck/tests/functional/selenium/webhook/WebhooksSpec.groovy

```diff
@@ -36,4 +41,183 @@ class WebhooksSpec extends SeleniumBase {
         deleteProject(projectName)
     }
 
+    /**
+     * Create webhook in a project
+     */
+    void "create project with webhook"() {
+        given:
+        String projectName = 'CreateWebhookProject'
+        String webhookName = 'CreateWebhook'
+        setupProject(projectName)
+        WebhooksPage webhookPage = page WebhooksPage
+        LoginPage loginPage = go LoginPage
+        HomePage homePage = page HomePage
+
+        when:
+        loginPage.login(TEST_USER, TEST_PASS)
+        homePage.validatePage()
+        webhookPage.loadPageForProject(projectName)
+        createSimpleWebhook(webhookPage, webhookName)
```

Comment:
```suggestion
        createSimpleWebhook(webhookPage, webhookName)
        then: "checks for the WH to exist"
        containsName(webhookPage, webhookName)
        when: "remove the wh"
```
this way the validation is more explicit and easy to read, like after deleting the wh